Casks/3/3dgenceslicer.rb

```diff
@@ -9,9 +9,8 @@
 
   livecheck do
     url "https://support.3dgence.com/software.html"
-    regex(
-      %r{href=.*?/3DGence(?:[._-]|%20)?Slicer(?:[._-]|%20)(\d+(?:\.\d+)+)(?:[._-]|%20)\(v?\.?(\d+(?:\.\d+)+)\)\.dmg}i,
-    )
+    regex(/3DGence(?:[._-]|\s)?Slicer(?:[._-]|\s)?(\d+(?:\.\d+)+)(?:[._-]|\s)?
+          \((?:version|v)?(?:[._-]|\s)?(\d+(?:\.\d+)+)\)(?:[._-]|\s+)*system(?:[._-]|\s)MacOS/ix)
```

Comment:
```suggestion
    regex(/3DGence\s+SLICER\s+v?(\d+(?:\.\d+)+)\s+\((?:version\s+|v)?(\d+(?:\.\d+)+)\)\s+-\s+system\s+MacOS/im)
```

I think a simpler version with respect to whitespace is probably fine. I usually don't see whitespace turn into something like `[._-]` in loose text like this (filenames are a different story) but I have seen regexes break from using a literal space or `\s` instead of `\s+` (or `\s*`, when appropriate).